build.psm1

```diff
@@ -1194,17 +1194,50 @@ esac
         chmod 755 "$Staging/$Name" # only the executable should be executable
     }
 
-    $libunwind = switch ($Type) {
-        "deb" { "libunwind8" }
-        "rpm" { "libunwind" }
-    }
-
-    $libicu = switch ($Type) {
-        "deb" {
-            if ($IsUbuntu14) { "libicu52" }
-            elseif ($IsUbuntu16) { "libicu55" }
+    # Setup package dependencies
+    # These should match those in the Dockerfiles, but exclude tools like Git, which, and curl
+    $dependencies = if ($IsUbuntu) {
```

Comment:
Oh. Sorry, I just realized you pushed a newer version (perhaps based on my prodding). Yes, the newer version is much clearer, thanks!